packages/core/src/types.ts

```diff
@@ -50,6 +50,9 @@ export interface LottieViewProps {
    * represents the normalized progress of the animation. If you update this prop, the
```

Comment:
I'd rather just remove the reference to Animated number at the beginning of this instead of adding this clarification. The internals of `createAnimatedComponent` should not be something this library points to. This is just a remnant of V5's behaviour. The Sample in doc is enough.